src/cephadm/cephadm

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
 
```

Comment:
@liewegas 
On FreeBSD is that nog going to work.
Not that I have invested any time into checking if cephamd would fit into FreeBSD...
```
[~] wjw@cephdev.digiware.nl> which !$
which python3
/usr/local/bin/python3
```

Reply:
Well, this is going to be a very minor problem when porting this to BSD.